### PR TITLE
fix(examples): Improvements after tests

### PR DESCRIPTION
**Pull request by sebastianczech**
## Description

PR delivers improvements in `common_vmseries` and `common_vmseries_autoscale` examples:
- added VNET peering module
- changed PAN-OS to 10.2.8 (as 10.2.3 has problems with expired certificates and don't connect to Panorama)
- added rules in test infrastructure to allow communication from bastion host
- fixed issue with bootstrap options
- updated README

## Motivation and Context

#3 

## How Has This Been Tested?

Code was tested by deploying examples in the lab.

## Types of changes

- Bug fix (non-breaking change which fixes an issue)

## Checklist

- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes if appropriate.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by michalbil — APPROVED** [review]

File: examples/common_vmseries/README.md [hunk 1]
```diff
@@ -361,6 +363,7 @@ map(object({
 
 
 
+
```
[the remark this answers is not in the document]
Comment (sebastianczech) [reply examples/common_vmseries/README.md:366]: README.md is generated automatically according to https://github.com/PaloAltoNetworks/terraform-azurerm-swfw-modules/blob/refactor-modules/.terraform-docs.yml.

Are the new lines in generated in README.md a problem, if while viewing the file we don't see them. If it's a problem, then we need to fix template https://github.com/PaloAltoNetworks/terraform-azurerm-swfw-modules/blob/refactor-modules/.terraform-docs.yml . Should I open new issue for that ?

[the remark this answers is not in the document]
Comment (acelebanski) [reply examples/common_vmseries/README.md:366]: As Sebastian mentioned, it's automatically generated. And as long as it looks good in Preview (and apparently it is), I'd leave it alone.

**Comment by acelebanski** [comment]
Hi @sebastianczech, I committed a few small changes to this PR:
- Changed the `vnet_peering` module place in the glue code after `vnet` module
- Corrected description of `vnet_peering` variable
- Changed example tfvars for `vnet_peering` to reflect the actual `standalone_panorama` example values
- Changed names and order of NSG entries in `test_infrastructure` map in tfvars

Btw, I see you bumped PAN-OS version to 10.2.8. According to [current guidelines](https://theloop.paloaltonetworks.com/loop/support-software-release-guidance-vm-series-and-virtualization), the preferred version for Azure now is `10.2.7-h3` but `10.2.8` is the closest we can get I think...

File: examples/common_vmseries/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (michalbil) [reply examples/common_vmseries/README.md:366]: Yes, this is a template issue, I've made some adjustments there:
https://github.com/PaloAltoNetworks/terraform-azurerm-swfw-modules/pull/40

[the remark this answers is not in the document]
Comment (sebastianczech) [reply examples/common_vmseries/README.md:366]: thank you @michalbil

**Comment by sebastianczech** [comment]
> Hi @sebastianczech, I committed a few small changes to this PR:
> 
> * Changed the `vnet_peering` module place in the glue code after `vnet` module
> * Corrected description of `vnet_peering` variable
> * Changed example tfvars for `vnet_peering` to reflect the actual `standalone_panorama` example values
> * Changed names and order of NSG entries in `test_infrastructure` map in tfvars
> 
> Btw, I see you bumped PAN-OS version to 10.2.8. According to [current guidelines](https://theloop.paloaltonetworks.com/loop/support-software-release-guidance-vm-series-and-virtualization), the preferred version for Azure now is `10.2.7-h3` but `10.2.8` is the closest we can get I think...

thank you @acelebanski 
should I merge that PR or improvements for other examples you want to put in this PR ?

**Review by acelebanski — APPROVED** [review]
Brought dedicated & gwlb examples up to speed with the changes in common. I think it's fine to merge now. 👍

**Pull request merged**